scripts/single-node.sh

```diff
@@ -63,6 +63,9 @@ createGenesis() {
     # If you encounter: `sed: -I or -i may not be used with stdin` on MacOS you can mitigate by installing gnu-sed
     # https://gist.github.com/andre3k1/e3a1a7133fded5de5a9ee99c87c6fa0d?permalink_comment_id=3082272#gistcomment-3082272
 
+    # Override the default genesis.json consensus params version from 0 to 4.
+    sed -i'.bak' 's#"app": "0"#"app": "4"#g' "${APP_HOME}"/config/genesis.json
```

Comment:
shouldn't this be changed in the app where we set the default to be the latest version?

Reply:
It could be. Unfortunately the genesis file creation uses code from our SDK fork and celestia-core fork so it wasn't easy to make that change. I think we'd have to update the DefaultConsensusParams in celestia-core _and_ cosmos-sdk and then we'd have to do that repeatedly every time we bumped app versions. Alternatively we could override it in default_overrides.go but I don't think that will impact the genesis file.